lib/dbservice_web/controllers/group_session_controller.ex

```diff
@@ -139,4 +142,65 @@ defmodule DbserviceWeb.GroupSessionController do
       |> render("show.json", group_session: group_session)
     end
   end
+
+  swagger_path :get_auth_group_from_session do
+    get("/api/group-session/session-auth-group")
+    description("Fetches the associated auth group data for a session")
+
+    parameters do
+      session_id(:query, :integer, "The ID of the session", required: true)
```

Comment:
why will it be integer.. we usually get string session id

Reply:
ah it seems it will be an integer only.. just checked frontend code. we send ID instead of session_id (from session table)